pytd/tests/test_writer.py

```diff
@@ -162,6 +164,16 @@ def test_write_dataframe_overwrite(self):
         args, kwargs = self.table.client.api_client.create_bulk_import.call_args
         self.assertEqual(kwargs.get("params"), None)
 
+    def test_write_dataframe_msgpack(self):
+        self.writer.write_dataframe(
+            pd.DataFrame([[1, 2], [3, 4]]), self.table, "overwrite", fmt="msgpack"
+        )
+        api_client = self.table.client.api_client
+        self.assertTrue(api_client.create_bulk_import.called)
+        self.assertTrue(api_client.create_bulk_import().upload_part.called)
+        api_client.create_bulk_import().upload_part.assert_called_with("part", ANY, 62)
```

Comment:
It might be better to describe how you obtained `62` as a comment, or, if I understand correctly:

```py
# create msgpack.gz for pd.DataFrame([[1, 2], [3, 4]])
size =  # calculate size of the msgpack file
api_client.create_bulk_import().upload_part.assert_called_with("part", ANY, size)
```

Reply:
indeed. made a change for it fbb4a2f